ui/components/GrafanaComponent.js

```diff
@@ -202,7 +200,7 @@ class GrafanaComponent extends Component {
         }, self.handleError('There was an error communicating with Grafana'));
       }
 
-      handleError = (msg) => (error) => {
```

Comment:
Don't think entirely removing the nested function is right. It might be better if we did something like
`handleError = msg => () => {...`

Reply:
A single function is better than empty nested arrows.

Reply:
I'm afraid it isn't how it works. I've confirmed it is a higher-order function and needs to be this way. 
See: https://github.com/layer5io/meshery/blob/c28a5355ec372568d72fca3718efd685160191f3/ui/components/GrafanaSelectionComponent.js#L170

Reply:
https://github.com/layer5io/meshery/blob/c28a5355ec372568d72fca3718efd685160191f3/ui/components/GrafanaSelectionComponent.js#L170
`error` is passed in `msg` arg & it is working as expected. You may even check on your local and confirm there is no need for nested function.
Have a look at ss 👍 
<img width="607" alt="Screenshot 2020-05-01 at 3 49 07 AM" src="https://user-images.githubusercontent.com/44091822/80764650-cebe8300-8b5e-11ea-8889-67e080973267.png">

If you think there is some error, please let me know